policy/modules/system/logging.te

```diff
@@ -524,7 +524,9 @@ ifdef(`init_systemd',`
 	allow syslogd_t self:netlink_audit_socket connected_socket_perms;
 	allow syslogd_t self:capability2 audit_read;
 	allow syslogd_t self:capability { chown setgid setuid sys_ptrace };
-	allow syslogd_t self:netlink_audit_socket { getattr getopt read setopt write };
+	allow syslogd_t self:netlink_audit_socket { getattr getopt read setopt write nlmsg_write };
+
+	logging_set_loginuid(syslogd_t)
```

Comment:
This doesn't seem to make sense. It isn't a login program and it's not a user.

Reply:
I suspect that `audit2allow -R` was being used here and in other places before. It probably returned a less than optimal interface in absence of a better alternative. Scary, I recently actually suggested that we remove that functionality for audit2allow as it is proven counter productive.

Reply:
Instead of completely removing it, maybe have it suggest appropriate macros for permission sets.
E.g. `allow a_t b_t:file read_file_perms` instead of `allow a_t b_t:file { open read }`.
This would mirror some of the functionality of selint.

Reply:
I'm not using audit2allow. I removed the logging_set_loginuid() call for now; and I'll investigate it.